geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/geotools/GeoMesaDataStore.scala

```diff
@@ -547,9 +547,12 @@ abstract class GeoMesaDataStore[DS <: GeoMesaDataStore[DS]](val config: GeoMesaD
     }
     sft.getAttributeDescriptors.asScala.foreach { d =>
       if (indexed(d)) {
-        val fields = Seq(d.getLocalName) ++ Option(sft.getGeomField) ++ sft.getDtgField
-        val existing = sft.getIndices
-        if (!existing.exists(e => e.name == AttributeIndex.name && e.attributes.map(remapCol) == fields)) {
+        val existing = {
```

Comment:
this is the only logic change

Reply:
Were updates removing indexes? :grimacing:

Reply:
:sweat_smile: